mutiny-core/src/nodemanager.rs

```diff
@@ -356,9 +364,16 @@ impl NodeManager {
 
         let node_storage = storage.get_nodes()?;
 
+        // Remove the archived nodes, we don't need to start them up.
+        let unarchived_nodes = node_storage
+            .clone()
+            .nodes
+            .into_iter()
+            .filter(|(_, n)| !n.is_archived());
```

Comment:
We may want to apply this filter on all of the calls we do in node manager where we look through all the nodes.

Reply:
Wait, nvm, we only check the node list so as long as unarchived nodes are not added to the node list, then it is fine.